ee/codegen/src/generators/node-output.ts

```diff
@@ -0,0 +1,80 @@
+import { python } from "@fern-api/python-ast";
+import { AstNode } from "@fern-api/python-ast/core/AstNode";
+import { Writer } from "@fern-api/python-ast/core/Writer";
+
+import { WorkflowContext } from "src/context";
+import { GenericNodeContext } from "src/context/node-context/generic-node";
+import { WorkflowValueDescriptor } from "src/generators/workflow-value-descriptor";
+import { NodeOutput as NodeOutputType } from "src/types/vellum";
+import { getVellumVariablePrimitiveType } from "src/utils/vellum-variables";
+
+export declare namespace NodeOutput {
+  export interface Args {
+    nodeOutputs: NodeOutputType[];
+    nodeContext: GenericNodeContext;
+    workflowContext: WorkflowContext;
+  }
+}
+
+export class NodeOutput extends AstNode {
```

Comment:
[nit] I'd make this class and file plural, since we are generating the full class